src/components/Card.js

```diff
@@ -1,32 +1,62 @@
 class Card {
-  constructor(data, cardSelector, handleCardClick) {
+  constructor(
```

Comment:
The trash button should on only your cards, because you can't delete not own cards